### PR TITLE
Update util.py

### PR DESCRIPTION
**Pull request by redsector72**
Referring to  #3202 and #3105 and #2824 I found out the Blackslist is really buggy.
Here is my "workaround" patch for #3202.
Maybe someone else will dig into it and discover why it is not working at all.
I had trouble finding the "log directory" but my knowledge is limited. this version will work with "multi=true"

### CODE REVIEW

**Comment by timgrossmann** [comment]
@redsector72 Thank you very much for your contribution.
I seldomly see people really use this feature so I'm sorry you're having issues.

Could you explain what goes wrong or what the wrong behaviour looks like, please?

**Comment by redsector72** [comment]
Sure.
The blacklist is written but according to my experience (python 2.7) it is never referenced. Maybe it works with higher version of python as is, i can't check this.

I've implemented the check if an user is in the blacklist in the user check functions. As is it works on python 2.7. If the user is in the blacklist it return false and there is not "like" action.

Now I see the "already liked" dropping in my stats, as expected.

File: instapy/like_util.py [hunk 1]
```diff
@@ -558,8 +558,10 @@ def like_image(browser, username, blacklist, logger, logfolder):
     if quota_supervisor("likes") == "jump":
         return False, "jumped"
 
-    like_xpath = "//button/span[@aria-label='Like']"
-    unlike_xpath = "//button/span[@aria-label='Unlike']"
+#    like_xpath = "//button/span[@aria-label='Like']"
+#    unlike_xpath = "//button/span[@aria-label='Unlike']"
+    like_xpath = "//button/span[@class='glyphsSpriteHeart__outline__24__grey_9 u-__7' and @aria-label='Like']"
```
Comment (timgrossmann) [inline instapy/like_util.py]: Could you help me with this part?
Is this a special selector for the elements you need?
They are too specific, what is the problem with the ones you commented out?

If you replace selectors, please remove the old ones and don't comment them out.

Thanks 😊

Comment (redsector72) [reply instapy/like_util.py]: This part should not be in the Commit. 
I'm not really good with GIT.
Anyway this part is to fix the like button change in the new update, like #3229

**Comment by redsector72** [comment]
I'm sorry I'm not able to remove the commit from like_util.
Can some one leave only the one for util or I have to make a new pull request?

**Comment by timgrossmann** [comment]
@redsector72 Sorry for that, I think creating a new one would be better because all the commits will appear in the commit history.

Hope that's ok for you.
Thank you

**Pull request closed without merging**